blocks/header/header.css

```diff
@@ -735,3 +735,23 @@ li
     display: none;
   }
 }
+
+.product-discovery--search-bar-results__grid {
+    padding: var(--spacing-xsmall);
+}
+
+.product-discovery--search-bar-results__outer-container .product-discovery--search-bar-results__grid {
+    box-shadow: 2px 2px 2px lightgray;
+    border: 1px solid lightgray;
```

Comment:
Is shadow meant to be just on the right, and to overlap slightly with the view all button? I changed shadow to red to show the issue. IMO we should stop adding opinionated CSS to boilerplate. See below comment.

![image](https://github.com/user-attachments/assets/d294653a-964f-460f-9736-ecfc0bf691e2)